### ISSUE TITLE
Wrong structurize version?

### ISSUE THREAD

**Issue by SirVapenation** [ldtteam/minecolonies#6609]
Good morning/afternoon!

Attached is the error in question that minecraft tells me:
![Capture](https://user-images.githubusercontent.com/79653720/109179574-0afb4980-7758-11eb-86a0-7fc4842b03c5.JPG)

When looking at the log this is what stands out:
-- MOD minecolonies --
Details:
	Mod File: minecolonies-0.13.437-RELEASE-universal.jar
	Failure message: Mod minecolonies requires structurize 0.13.96-ALPHA
		Currently, structurize is 0.13.100-RELEASE
	Mod Version: 0.13.437-RELEASE
	Mod Issue URL: https://github.com/ldtteam/minecolonies/issues/new
	Exception message: MISSING EXCEPTION MESSAGE

Here is the full Crash log:
[crash-2021-02-25_10.47.05-fml.txt](https://github.com/ldtteam/minecolonies/files/6044158/crash-2021-02-25_10.47.05-fml.txt)

Basically, the Wand mod is outdated both on the website and on Curseforge launcher.

**Comment by SirVapenation** [comment]
Bit of an update, i figured out the issue and solved it.
Yes, Structurize was outdated (or over-dated, i dont know)
I was able to download the correct version.

**Comment by ravenbuilder934** [comment]
for future reference, you should be using the latest alpha of minecolonies and the second-latest alpha of structurize

**Comment by SirVapenation** [comment]
> for future reference, you should be using the latest alpha of minecolonies and the second-latest alpha of structurize

Yeah, sorry. The issue was it was on Curseforge, so i expected it to be the right one
